app/views/invoice_mailer/invoice_email.text.erb

```diff
@@ -1,6 +1,10 @@
-Invoice sent to you by <%= @organization.name %>
-=====================================================
+<% if @text %>
+  <%= @text %>
```

Comment:
For this to be functional, I think we need to enable the user to use some variables.

At least who made the invoice.